adapters/src/main/java/org/jboss/jca/adapters/jdbc/local/LocalManagedConnectionFactory.java

```diff
@@ -104,9 +104,6 @@ public Object createConnectionFactory(ConnectionManager cm) throws ResourceExcep
       if (dataSourceClass == null && connectionURL == null && driverClass != null)
          throw new ResourceException(bundle.connectionURLNull());
 
-      if (dataSourceClass != null && connectionProps.size() == 0)
-          throw new ResourceException(bundle.nonConnectionPropertyDefinedForDatasource(dataSourceClass));
-
```

Comment:
This check is intended, because if the `dataSourceClass` is defined, IronJacamar will use `javax.sql.DataSource.getConnection()` to create the physical db connection, the `<connection-property>` is required in this case.